readthedocs/templates/projects/notifications/deprecated_config_file_used_email.html

```diff
@@ -1,25 +1,27 @@
 {% extends "core/email/common.html" %}
 {% block content %}
 The Read the Docs build system will start requiring a configuration file v2 (<code>.readthedocs.yaml</code>) starting on <strong>September 25, 2023</strong>.
-We are scheduling brownout days to provide extra reminders by failing build without a configuration file v2 during some hours before the final day.
-Keep these dates in mind to avoid unexpected behaviours:
+We are sending weekly notifications about this issue to all impacted users,
+as well as temporarily failures (brownouts) as the date approaches for those who haven't migrated their projects.
+
+The timeline for this deprecation is as follows:
 
 <ul>
     <li><strong>Monday, July 24, 2023</strong>: Do the first brownout (temporarily enforce this deprecation) for 12 hours: 00:01 PST to 11:59 PST (noon)</li>
     <li><strong>Monday, August 14, 2023</strong>: Do a second brownout (temporarily enforce this deprecation) for 24 hours: 00:01 PST to 23:59 PST (midnight)</li>
-    <li><strong>Monday, September 4, 2023</strong> Do a third and final brownout (temporarily enforce this deprecation) for 48 hours: 00:01 PST to 23:59 PST (midnight)</li>
+    <li><strong>Monday, September 4, 2023</strong> Do a third and final brownout (temporarily enforce this deprecation) for 48 hours: 00:01 PST to September 5, 2023 23:59 PST (midnight)</li>
     <li><strong>Monday, September 25, 2023</strong>: Fully remove support for building documentation without configuration file v2.</li>
 </ul>
 
-We have identified the following projects where you are admin are impacted by this deprecation:
+We have identified the following projects where you a maintainer, and are impacted by this deprecation:
 
 <ul>
 {% for project in object %}
-    <li>{{ project.slug }}</li>
+    <li><a href="{{ production_uri }}{% project.get_absolute_url %}">{{ project.slug }}</a></li>
 {% endfor %}
 </ul>
 
-You require to add a configuration file to your projects to ensure they continues building successfully and stop receiving these notifications.
+Please add configuration file to your projects to ensure they continue building successfully and stop receiving these notifications.
```

Comment:
Missing a word here or a pluralization issue perhaps. This maybe makes sense, though is plural when only one project is listed above:

```suggestion
Please add configuration files to your projects to ensure they continue building successfully and stop receiving these notifications.
```

Reply:
```suggestion
Please add a configuration file to your projects to ensure they continue building successfully and stop receiving these notifications.
```

Reply:
```suggestion
Please add a configuration file to your projects to ensure that they continue building successfully and to stop receiving these notifications.
```